src/Json.Schema.ToDotNet/ClassGenerator.cs

```diff
@@ -207,6 +218,27 @@ private MemberDeclarationSyntax GenerateValueComparerProperty()
                 .WithSemicolonToken(SyntaxFactory.Token(SyntaxKind.SemicolonToken));
         }
 
+        private MemberDeclarationSyntax GenerateComparerProperty()
```

Comment:
>GenerateComparerProperty [](http://example.com/codeflow?start=40&length=24)

This code uses Roslyn to emit a property in the file like this:

public static IComparer<Address> Comparer => AddressComparer.Instance;

following the existing patter we use for IEqualityComparer.